draft-ietf-teep-architecture.md

```diff
@@ -1329,23 +1329,47 @@ may need to be updated.
 ## Compromised TAM
 
 Device TEEs are responsible for validating the supplied TAM certificates
-to determine that the TAM is trustworthy. A compromised TAM can install
-arbitrary TAs to, potentially, many devices, which makes TAMs high value
-targets. An advanced, persistent threat actor could potentially compromise
-a TAM without impacting its certificate or raising concern from the TAM's
-operator. To mitigate this threat, TEEs have several options for 
-detecting and mitigating a compromised TAM:
+to determine that the TAM is trustworthy. A compromised TAM may bring multiple threats 
+and damage to user devices that it can managee and the device owners. 
+In one hand, the device information of many devices that the TAM manages and 
+therefore have in its system may be leaked to a bad actor. On the other hand,
+a compromised TAM can install many TAs to launch somee DoS attack to devices,
+for example, by filling up a device's TEE resources reserved for TAs such that 
+other TAs may not get resource to be installed or properly function. It may
+also install malicious TAs to, potentially, many devices, if it can also 
+get or access a Trusted Component signer key that is trusted by the TEEs. 
+This makes TAMs high value targets. An advanced, persistent threat actor 
+could potentially compromise a TAM without impacting its certificate or 
+raising concern from the TAM's operator.
+
+To mitigate this threat, TEE Agents have several options as listed below but 
+not limit to these for detecting and mitigating a compromised TAM:
 
 1. Apply an ACL to the TAM, indicating which TAs it is permitted to install.
 2. Use a transparency log to expose a TA compromise: TAMs publish an 
 out-of-band record of TA releases, allowing a TEE to cross-check the TAs
 delivered against the TA releases in order to detect a TAM compromise.
 3. Use remote attestation of the TAM to prove trustworthiness.
 
-While TAs are signed by the creator and delivered by the TAM, there is
-still a threat present in allowing a potentially compromised TAM to install
-any TA it wants; for example, a known-compromised TA or an explicitly
-malicious TA.
+## Hostile TAM
+
+There are also threats of hostile or abusive TAMs that act out of expectation 
+of Device Administrators, for example, pushing out TAs that contain 
+some data collection or use users' device resources for distributed jobs for 
+a TAM. All the potential damanges that a compromised TAM can make can be done by a
+hostile TAM. A hostile TAM differs in knowingly launch some abusive behavior.
+
```

Comment:
The main case above is if someone can tamper with the trust anchor store, whether locally or via a compromised TAM inserting a hostile TAM's key.   The mitigation there are trust anchor store protection as discussed in section 12.4, and some secure auditing of changes to the trust anchor store.
(Maybe add the above in text.)

Reply:
The tampering of Trust Anchor store brings a different use case. It is a source of injecting a new hostile TAM rather than an existing one that changed behavior. Will mention this.